recipe/meta.yaml

```diff
@@ -254,30 +308,29 @@ outputs:
         - python >=3.10
         - pip
         - cython <3
-        - custatevec  ={{ cusv_version }}
-        - cutensornet ={{ cutn_version }}
+        - custatevec  =1.7.0
+        - cutensornet =2.6.0
+        - cudensitymat =0.0.5
         - packaging
       run:
         - python >=3.10
-        - numpy  >=1.21,<2
+        - numpy  >=1.21
         - cupy   >=13.0
         - {{ pin_subpackage('custatevec', max_pin='x') }}
         - {{ pin_subpackage('cutensornet', max_pin='x') }}
-        - cuda-version >=11.0.*,<13.a0  # again, cuQuantum Python is "CUDA agnostic"
+        - {{ pin_subpackage('cudensitymat', max_pin='x') }}
+        - cuda-version >={{ cuda_major }}.0,<{{ cuda_major + 1 }}
       run_constrained:
         - pytorch >=1.10
         - mpi4py >=3.1.0
-        # we don't have to pin Qiskit or Cirq here because their versions are fairly recent on conda-forge
     test:
       requires:
-        - cuda-driver-dev  # [linux and (cuda_compiler_version or "").startswith("12")]
+        - cuda-driver-dev  # [linux and cuda_compiler_version in ("12.0", "12.6")]
+      files:
+        - ./common/*
+        - ./cuquantum-python/*
       commands:
-        # perform the import test via CLI because we need to load the stub for CUDA 12
-        - export CUDA_STUB="$PREFIX/lib/stubs/libcuda.so"  # [(cuda_compiler_version or "").startswith("12")]
-        - LD_PRELOAD="$CUDA_STUB" python -c "import cuquantum"
-        - LD_PRELOAD="$CUDA_STUB" python -c "import cuquantum.custatevec"
-        - LD_PRELOAD="$CUDA_STUB" python -c "import cuquantum.cutensornet"
-        - LD_PRELOAD="$CUDA_STUB" python -m cuquantum --includes --libs --target custatevec --target cutensornet
+        - ./cuquantum-python/cuquantum-python-tests
     about:
```

Comment:
`pip check` fails here due to the way the package is installed (see `recipe/cuquantum-python/cuquantum-python-install`)
```
pip install --no-deps --no-build-isolation -vvv .
```

Reply:
That's not a reason why `pip check` would fail. `--no-deps` just stops pip from installing dependencies. The dependencies should be installed by conda instead. 

The reason you would want to skip `pip check` is if the python metadata requires python packages whose names do not match the corresponding conda packages or are not python packages. e.g. nvidia-cublas-cu11 (libcublas) or cutensornet-cu12 (cutensornet).

Reply:
Thanks, that makes sense. Here both apply, so we'll need to skip `pip check`.

Reply:
Resolving since `cuquantum-python` output will be removed.